frontend/src/cljs/imas_seamap/db.cljs

```diff
@@ -47,8 +47,16 @@
                      :left-drawer        false
                      :right-drawer       false
                      :drawer-panel-stack []
-                     :catalogue          {:tab      "org"
-                                          :expanded #{}}
+                     :catalogue          {:habitat     {:tab "org"     ; NOTE: Move away from map to vector to support arbitrary number of tabs/groups?
+                                                        :expanded #{}}
+                                          :boundaries  {:tab "org"
+                                                        :expanded #{}}
+                                          :bathymetry  {:tab "org"
+                                                        :expanded #{}}
+                                          :imagery     {:tab "org"
+                                                        :expanded #{}}
+                                          :third-party {:tab "org"
+                                                        :expanded #{}}}
```

Comment:
This is the obvious bit that stands out -- remember we'll have an arbitrary number of categories soon, not just this 5.  It's fine if you're remembering that and just want to do one thing at a time (I encourage that, even!), but do keep it in mind.
This is probably more relevant below where you've updated the app-state spec (thank you!); this is good, but you won't be able do that shortly, so it might be worth just making this section a s/map-of or whatever the syntax is, instead of speccing every key.